### PR TITLE
added notifier queue

### PR DESCRIPTION
**Pull request by syed-awais-ali**
Configuration Pull Request
---

Make sure that the following steps are done before merging:

  - [ ] A DevOps team member has approved the PR if it is code shared across multiple services and you don't own all of the services.
  - [ ] Are you adding any new default values that need to be overridden when this change goes live? If so:
    - [ ] Update the appropriate internal repo (be sure to update for all our environments)
    - [ ] If you are updating a secure value rather than an internal one, file a DEVOPS ticket with details.
    - [ ] Add an entry to the CHANGELOG.
  - [ ] If you are making a complicated change, have you performed the proper testing specified on the [Ops Ansible Testing Checklist](https://openedx.atlassian.net/wiki/display/EdxOps/Ops+Ansible+Testing+Checklist)?  Adding a new variable does not require the full list (although testing on a sandbox is a great idea to ensure it links with your downstream code changes).

### CODE REVIEW

File: playbooks/roles/ecomworker/defaults/main.yml [hunk 1]
```diff
@@ -38,6 +38,7 @@ ECOMMERCE_WORKER_BROKER_TRANSPORT: 'amqp'
 # Default broker URL. See http://celery.readthedocs.org/en/latest/configuration.html#broker-url.
 ECOMMERCE_WORKER_BROKER_URL: '{{ ECOMMERCE_WORKER_BROKER_TRANSPORT }}://{{ ECOMMERCE_WORKER_BROKER_USERNAME }}:{{ ECOMMERCE_WORKER_BROKER_PASSWORD }}@{{ ECOMMERCE_WORKER_BROKER_HOST }}:{{ ECOMMERCE_WORKER_BROKER_PORT }}'
 ECOMMERCE_WORKER_CONCURRENCY: 4
```
Comment (feanil) [inline playbooks/roles/ecomworker/defaults/main.yml]: Where is this setting used?  How will this override the default queue name?

File: playbooks/roles/notifier/defaults/main.yml [hunk 2]
```diff
@@ -130,3 +131,4 @@ notifier_env_vars:
   FORUM_DIGEST_TASK_RATE_LIMIT: "{{ NOTIFIER_FORUM_DIGEST_TASK_RATE_LIMIT }}"
   DEAD_MANS_SNITCH_URL: "{{ NOTIFIER_DEAD_MANS_SNITCH_URL }}"
   BOTO_CONFIG: "{{ notifier_app_dir }}/.boto"
+  NOTIFIER_ROUTING_KEY: "{{ NOTIFIER_CELERY_QUEUE }}"
```
Comment (feanil) [inline playbooks/roles/notifier/defaults/main.yml]: I don't see anything that uses this key, is there a corresponding change to the notifier code?

Comment (syed-awais-ali) [reply playbooks/roles/notifier/defaults/main.yml]: [notifier](https://github.com/edx/notifier/pull/73) will consume this.

File: playbooks/roles/ecomworker/defaults/main.yml (hunk 1 above)
Comment (coryleeio) [reply playbooks/roles/ecomworker/defaults/main.yml]: The consumption should be here:
https://github.com/edx/ecommerce/pull/2129

Once the PR is updated.

Comment (coryleeio) [reply playbooks/roles/ecomworker/defaults/main.yml]: @syed-awais-ali Can you separate out the ecommerce bit into a different PR, that will make reverting simpler and less disruptive if need be and let you focus on landing the notifier PR

**Review by coryleeio — CHANGES_REQUESTED** [review]

**Review by coryleeio — APPROVED** [review]

**Review by coryleeio — APPROVED** [review]

**Pull request closed without merging**